inspectit-oce-agent/src/system-test/java/rocks/inspectit/oce/metrics/GCMetricsSysTest.java

```diff
@@ -38,6 +40,11 @@ public void testGCPauseCapturing() throws Exception {
             System.gc();
             System.runFinalization();
             if (objRef.get() != null) {
+                // even more trash
```

Comment:
what's we the weak reference idea?

Reply:
should we also decrease the default metrics collection frequency for the system tests (I am not sure if this has been done, but could this also be a problem here)... Default is 15s?

Reply:
The GC metrics recorder is not polling based, therefore the frequency does not impact it. The recorder performs its duty based on events sent by the JVM after a GC.

Reply:
fancy